org.mixedrealitytoolkit.standardassets/CHANGELOG.md

```diff
@@ -2,8 +2,8 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
-## [3.0.2-development] - 2024-02-04
+## [3.1.0-development] - 2024-02-09
 
 ### Fixed
 
-* Fixed support for UPM package publishing in the Unity Asset Store.
```

Comment:
Please keep this line

`* Fixed support for UPM package publishing in the Unity Asset Store.`

Never remove stuff from CHANGELOG.MD, only add.

I just added CHANGELOG's for the Unity Asset Store support, that is why it is pretty bare

Reply:
Done